Weather/src/app/factory/factoryClassTimeOfDay.ts

```diff
@@ -0,0 +1,34 @@
+import { TimeOfDayEnum } from "../enums/enumForTimeOfDay";
+
+export class factoryClassTimeOfDay {
```

Comment:
All classes start from the uppercase letter. It isn't a factory it seems to be a service or some model. You can go another way, like via model for view instead of the factory.

P.S: Example of the factory - https://refactoring.guru/design-patterns/factory-method/typescript/example